superset-frontend/src/explore/components/DatasourcePanel/DatasourcePanelItem.tsx

```diff
@@ -223,46 +221,34 @@ const DatasourcePanelItem: FC<Props> = ({ index, style, data }) => {
               flex-shrink: 0;
             `}
           >
-            {isColumnSection
-              ? t(`Showing %s of %s`, columnSlice?.length, totalColumns)
-              : t(`Showing %s of %s`, metricSlice?.length, totalMetrics)}
+            {t(`Showing %s of %s items`, item.showingItems, item.totalItems)}
```

Comment:
@justinpark restored `Showing x of x` (see recording in the summary to see how it works)